ruby-challenges.rb

```diff
@@ -22,7 +52,19 @@
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def remove_the_vowels(string)
+    string.gsub(/[aeiouAEIOUa]/, '')
+end
+
+# p remove_the_vowels beatles_album1 # --> "Rbbr Sl"
+# p remove_the_vowels beatles_album2 # --> "Sgt Pppr"
+# p remove_the_vowels beatles_album3 # --> "bby Rd"
+
 # Pseudo code:
+# Input: A string
+# Output: A string - same string but without vowels.
+# Create a method called remove_the_vowels with string as it's parameter.
+    # From the string, utilize method .gsub to search the string for any characters that are vowels [aeiouAEIOU] and replace it with '' (empty string). 
```

Comment:
great job!  You don't have to use regex, but excellently implemented.  the .delete method would also work. :)